Userland/Libraries/LibSQL/Value.cpp

```diff
@@ -222,7 +222,7 @@ bool Value::is_int() const
 ErrorOr<String> Value::to_string() const
 {
     if (is_null())
-        return String::from_utf8("(null)"sv);
+        return "(null)"_string;
```

Comment:
This one is `from_utf8` on purpose because SQLServer is one of the areas that will likely want to handle OOM